Gui/opensim/view/src/org/opensim/view/experimentaldata/ExperimentalMarkerSetNode.java

```diff
@@ -173,38 +173,38 @@ public Color getColor()
         return new Color((float)colorAsVec3.get(0), (float)colorAsVec3.get(1), (float)colorAsVec3.get(2));
     }
    
-    public void setMarkerScaleFactorUI(double newFactor) {
-        setMarkerScaleFactorUI(newFactor, true);
+    public void setMarkerRadiusUI(double newRadius) {
+        setMarkerRadiusUI(newRadius, true);
     }
     
-    void setMarkerScaleFactorUI(final double newFactor, boolean allowUndo)
+    void setMarkerRadiusUI(final double newRadius, boolean allowUndo)
     {
-        final double oldMarkerScaleFactor = getMarkerScaleFactor();
+        final double oldMarkerSize = getMarkerRadius();
         if (allowUndo){
             AbstractUndoableEdit auEdit = new AbstractUndoableEdit(){
                @Override
                public void undo() throws CannotUndoException {
                    super.undo();
-                   setMarkerScaleFactorUI(oldMarkerScaleFactor, false);
+                   setMarkerRadiusUI(oldMarkerSize, false);
                }
                @Override
                public void redo() throws CannotRedoException {
                    super.redo();
-                   setMarkerScaleFactorUI(newFactor, true);
+                   setMarkerRadiusUI(newRadius, false);
```

Comment:
What is the reason for changing `true` to `false`?

Reply:
This relates to your earlier reported bug with multiple undo/redo/undo, the change to false handles this properly by making the undo manager fully responsible for the undo/redo unwinding. Youcan test the latest artifcat and you should be able to verify. Since unrelated I can "undo" it as well.

Reply:
No it's fine I just mostly wanted to make sure  you intended this change.